pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -46,8 +46,9 @@ def connection_group():
 
     This command groupdefines commands to manage persistent WBEM connections
     that have a name. The connections are stored in a connections file named
-    'pywbemcliservers.json' in the current directory. The connection name can
-    be used as a shorthand for the WBEM server via the '--name' general option.
+    'pywbemcli_connections.yaml' in the current directory. The connection name
```

Comment:
Change "connection name" to "connection definition name", as agreed in PR #295.

Reply:
DONE